browser/js/common/directives/navbar/navbar.js

```diff
@@ -9,14 +9,46 @@ app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state,
 			scope.goToSection = GoToSectionFactory.goToSection;
 			scope.class = 'hide-menu';
 
+			scope.displayWhatIs = true;
+			scope.displayGetInvolved = true;
+			scope.displayResources = true;
+			scope.displayOurTeam = true;
+
+			// This toggle menu is used for the hamburger icon and when to
+			// display the titles of the dropdown menus
 			scope.toggleMenu = function () {
 				if (scope.class == 'hide-menu') {
 					scope.class = 'show-menu';
 				} else {
 					scope.class='hide-menu';
 				}
 			}
-			
+
+			// This toggle function is used for the mobile nav-bar lists
+			scope.toggle = function (val) {
+				if (val == "displayWhatIs") {
+					scope.displayWhatIs = !scope.displayWhatIs;
+					scope.displayGetInvolved = true;
+					scope.displayResources = true;
+					scope.displayOurTeam = true;
```

Comment:
so scope[scopeVar] accesses the value of the variable of the scope? And does that mean that angular just stores all variables we create in a controller inside this array?